static/css/disaster_dashboard.scss

```diff
@@ -15,3 +15,212 @@
  */
 
  @import "base";
+
+.all-disasters-legend-entry {
+  display: flex;
+  align-items: center;
+  padding: 0 0.5rem;
+}
+
+.all-disasters-legend-color {
+  width: 1rem;
+  height: 1rem;
+  margin-right: 0.25rem;
+}
+
+.all-disasters-legend {
+  display: flex;
+  padding: 1rem;
+}
+
+.disaster-dashboard-tab-section {
+  display: flex;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.125);
+}
+
+.disaster-dashboard-tab,
+.disaster-dashboard-tab-selected {
+  width: 30vw;
+  text-align: center;
+  padding: 1rem 0;
+  cursor: pointer;
+}
+
+.disaster-dashboard-tab-selected,
+.disaster-dashboard-tab:hover {
+  border-bottom: 1px solid var(--dc-primary);
+  color: var(--dc-primary)
+}
+
+#main-disaster-dashboard {
+  margin: 0;
+  padding: 2rem;
+}
+
+.disaster-dashboard-date-selector {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin-top: 1rem;
+}
+
+#disaster-dashboard-date-selector-input {
+  width: 12rem;
+  margin-left: 0.5rem;
+}
+
+.disaster-dashboard-map-section {
+  border: 1px solid rgba(0, 0, 0, 0.125);
```

Comment:
this could be a file variable (there are other opportunities as well but this is the most obvious one).

Reply:
done